bundle/regal/ast.rego

```diff
@@ -20,6 +20,7 @@ package_name := concat(".", [path.value |
 
 tests := [rule |
 	some rule in input.rules
+	not rule.head.args
```

Comment:
Does this warrant a comment to say that this is saying `if not function`?

Reply:
Yeah, I guess I've become a little blind to these things by now 😄 Perhaps even better to add a `is_function(rule)` helper to `ast.rego`? I can do so in a follow-up PR.

Reply:
💭 Or even
```rego
is_function(rule) if not rule.head.args
```

Reply:
Yeah that works better!